rfcs/0000-execute-builds-in-standard-containers.md

```diff
@@ -0,0 +1,132 @@
+## Problem
+
+Currently, kpack executes each stage of the [CNB Lifecycle][cnb-lifecycle] inside of
+individual [init containers][init-containers]. This has two major benefits:
+
+1. Different stages of the lifecycle are isolated from one another allowing the credentials required to push to the
```

Comment:
Perhaps, just as important, this prevents application code from from accessing the creds as well.